docs/postgres.md

```diff
@@ -61,7 +61,22 @@ Note that the PostgreSQL database *must* have the correct encoding set
 
 You may need to enable password authentication so `synapse_user` can
 connect to the database. See
-<https://www.postgresql.org/docs/11/auth-pg-hba-conf.html>.
+<https://www.postgresql.org/docs/current/auth-pg-hba-conf.html>.
+
+If you get an error like `FATAL:  Ident authentication failed for user "synapse_user"`, you may need to use an authentication method other than `ident`.
+
+If the `synapse_user` user has a password, add the password to the `database:` section of `homeserver.yaml`. Then add the following to `pg_hba.conf`:
```

Comment:
I don't think the formatting makes it entirely clear that these lines are only necessary if you've received the `Ident` error. I might format hba lines into a bullet point list, to make it clear that if you don't receive and `Ident` error, you can just skip to the next section.

Reply:
That's fine. I just wanted the information in there.

Reply:
Sounds good. Would you like me to add the changelog as well?

Reply:
Sure, add the changelog.